src/placeos-models/user.cr

```diff
@@ -164,12 +164,16 @@ module PlaceOS::Model
     secondary_index :email
 
     def self.find_by_email(authority_id : String, email : String)
-      User.where(email: email, authority_id: authority_id).first?
+      User.collection_query { |table|
+        table
+          .get_all([authority_id], index: :authority_id)
+          .filter(&.["email"].match("(?i)^#{email}$"))
+      }.first?
     end
 
     def self.find_by_emails(authority_id : String, emails : Array(String))
```

Comment:
I think the plural and singular methods should have the same semantics, otherwise, it'll be confusing.

I think using a `get_all` on `authority_id` in conjunction with [a map](https://rethinkdb.com/api/ruby/map/) should be able to achieve this.

May as well write `find_by_email` with `find_by_emails` too